src/components/card-list/card-list.component.jsx

```diff
@@ -5,17 +5,17 @@ import Card from '../card/card.component';
 import BackToTop from '../back-to-top/back-to-top.component';
 
 const CardList = ({ pokemon }) => (
-  <div>
-    <div className="card-list">
+  <>
```

Comment:
React Fragment https://reactjs.org/docs/fragments.html#short-syntax

Reply:
I've seen this before. I reorganized my block of code so the card-list div is the highest element. React fragment no longer necessary.